cachi2/core/package_managers/metayarn.py

```diff
@@ -0,0 +1,122 @@
+import logging
```

Comment:
Oh, and I think we should stuff this under a new package rather than a top-level module, IOW, make this a `meta/yarn.py` or `_meta/yarn.py`.